Formula/m-cli.rb

```diff
@@ -1,17 +1,20 @@
 class MCli < Formula
   desc "Swiss Army Knife for Mac OS X"
   homepage "https://github.com/rgcr/m-cli"
-  url "https://github.com/rgcr/m-cli/archive/v0.1.0.tar.gz"
-  sha256 "d28f9b4abdb783554eee01432ffb55dbd206f018b6825018f00fe722af799963"
+  url "https://github.com/rgcr/m-cli/archive/v0.1.1.tar.gz"
+  sha256 "66c4b158c1fdfa976a33c97069ae96c2428ca92f5a01b27430ef36930a267d53"
 
   bottle :unneeded
 
   def install
     prefix.install Dir["*"]
-    # Use absolute rather than relative path to plugins.
     inreplace prefix/"m" do |s|
+      # Use absolute rather than relative path to plugins.
       s.gsub! /^\[ -L.*|^\s+\|\| pushd.*|^popd.*/, ""
       s.gsub! /MPATH=.*/, "MPATH=#{prefix}"
+      # Disable options "update" && "uninstall", they must be handled by brew
+      s.gsub! /update_mcli \&\&.*/, "printf \"Try: brew update && brew upgrade m-cli \\n\" && exit 0"
+      s.gsub! /uninstall_mcli \&\&.*/, "printf \"Try: brew uninstall m-cli \\n\" && exit 0"
```

Comment:
Can you please submit these patches (or at least issues requesting this functionality) to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!